pkg/resources/helpers_test.go

```diff
@@ -21,6 +22,26 @@ const (
 	onAll
 )
 
+var (
+	awsBucketUrl, awsBucketUrlIsSet = os.LookupEnv("AWS_EXTERNAL_BUCKET_URL")
```

Comment:
I had a comment about naming these in other PR. It also applies here.

Reply:
I would merge it and change everything in the stages pr